packages/ia-components/sandbox/theatres/audio-1/audio-1.js

```diff
@@ -14,7 +15,9 @@ import PlayTriangle from './play-triangle.svg'
 // import JwNavImg from './jw-screenshot2.png'
 
 import AudioNavigation from './audio-nav'
-import TheatreHamburger from '../../theatre-hamburger/theatre-hamburger'
+// import TheatreHamburger from '../../theatre-hamburger/theatre-hamburger'
```

Comment:
Was the import wrong? if you are correcting it on line 20, then remove this commented line